platforms/platform-core/src/CoreRequest.ts

```diff
@@ -30,15 +41,17 @@ export class CoreRequest extends JovoRequest {
   }
 
   getEntities(): JovoInput['entities'] {
-    return this.input?.entities;
+    return this.input?.nlu?.entities || this.input?.entities;
```

Comment:
The root `entities` should be prioritized:

```
return this.input?.entities || this.input?.nlu?.entities;
```

Reply:
Got it!